ITR/data/base_providers.py

```diff
@@ -717,121 +821,91 @@ def project_ei_targets(self, company: ICompanyData, production_bm: pd.Series) ->
                     else:
-                        # Get the intensity data
-                        intensity_data = historic_data.emissions_intensities.__getattribute__(scope)
-                        last_year_data = next((i for i in reversed(intensity_data) if np.isfinite(i.value.magnitude)),
-                                              None)
-
-                    if last_year_data is None:  # No historic data, so no trajectory projections to use either
-                        ei_projection_scopes[scope] = None
-                        continue
-
-                    if base_year > last_year_data.year:
-                        trajectory_ei = projected_intensities.__getattribute__(scope).projections
-                        last_year_data = next((ei for ei in trajectory_ei if ei.year == base_year), None)
-                        warnings.warn(f"Emission intensity at base year for scope {scope} target for company "
-                                      f"{company.company_name} is estimated with trajectory projection.")
-
-                    last_year, value_last_year = last_year_data.year, last_year_data.value
+                        last_ei_year = target.target_base_year
+                        last_ei_value = Q_(target.target_base_year_qty, target.target_base_year_unit)
+
                     target_year = target.target_end_year
                     # Attribute target_reduction_pct of ITargetData is currently a fraction, not a percentage.
-                    target_value = Q_(target.target_base_year_qty * (1 - target.target_reduction_pct),
-                                      target.target_base_year_unit)
-                    CAGR = self._compute_CAGR(value_last_year, target_value, (target_year - last_year))
-                    ei_projections = [ICompanyEIProjection(year=year, value=value_last_year * (1 + CAGR) ** (y + 1))
-                                      for y, year in enumerate(range(1 + last_year, 1 + target_year))]
-                    if ei_projection_scopes[scope] is not None:
-                        ei_projection_scopes[scope].projections.extend(ei_projections)
-                    else:
-                        ei_projection_scopes[scope] = ICompanyEIProjections(projections=ei_projections,
-                                                                            ei_metric=IntensityMetric.parse_obj({'units': target.target_base_year_unit}))
+                    target_ei_value = Q_(target.target_base_year_qty * (1 - target.target_reduction_pct),
+                                         target.target_base_year_unit)
+                    CAGR = self._compute_CAGR(last_ei_value, target_ei_value, (target_year - last_ei_year))
+                    model_ei_projections = [ICompanyEIProjection(year=year, value=last_ei_value * (1 + CAGR) ** (y + 1))
+                                            for y, year in enumerate(range(last_ei_year, 1+target_year))
```

Comment:
enumerate was for `range(1 + last_year, 1 + target_year)`
we might have lost a year in this line change

Reply:
why is it the right thing to do?
original code had reasons, to exclude the initial year from calculation.
we now produce the different result due to a different range of years - where do we have a document or an article, which clarifies, what should be the right range in this context?

Reply:
I wrote the original code ;-). Initially all the target projections were done from data that had a consistent right-hand edge (typically 2020).  As we moved into 2022, by dribs and drabs, we got 2021 data.  The uneven data created NaNs when merged.  By starting the projection a year earlier we fill in the NaNs.  I also found a math error in the CAGR computation that needed to be fixed when we change the year window, so look out for that.  I found it by graphing the projections and seeing an anomaly.

Reply:
what i really mean is: where do we have a document or an article, which clarifies, what should be the right range in this context?
is ITR based on a methodology, or on opinions?

Reply:
In this particular case it's more about solving world problems than following or inventing science.  Whether a projection begins at base_year with base_production_value (using y==0 as the first exponent for the CAGR), or begins at base_year+1 with the first CAGR-reduced base_production_value (using y+1==1 as exponent instead of y==0), it does not change the math of either.  It's just a question of how conveniently we prepare data for downstream calculations.  I'll try to create a ragged left edge test case that shows the problem.